google-cloud-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -140,6 +150,7 @@ private LocalDatastoreHelper(Builder builder) {
     List<String> gcloudCommand = new ArrayList<>(Arrays.asList(GCLOUD_CMD_TEXT.split(" ")));
     gcloudCommand.add(GCLOUD_CMD_PORT_FLAG + "localhost:" + getPort());
     gcloudCommand.add(CONSISTENCY_FLAG + builder.consistency);
+    gcloudCommand.add(PROJECT_FLAG + getProjectId());
```

Comment:
I think this should only be set if the project id is not null

Reply:
Yeah, it seems confusing at first, but `getProjectId` will always return a non null value in this case, because we are passing a non null value to the `projectId` parameter in super constructor line 142 above..

Reply:
I've created a local variable now to eradicate the confusion.